utils/frame/sub-du/README.md

```diff
@@ -0,0 +1 @@
+# sub-du
```

Comment:
delete this file please.

Reply:
Maybe it's best to actually have a proper README?

Reply:
If you are keen on writing some readme please write it as module doc (`//!`) it will be more worthwhile and our CI will create the READM from it.